server/src/sessionManagement.mjs

```diff
@@ -59,6 +116,10 @@ class Session {
             // const sessionStartString = sessionStart.toISOString().replace(/T/, '_').replace(/\..+/, '');
             // logger.info(`${sessionStart.toISOString()}`);    
 
+            // Check if the output path is a WebSocket URL
+            const wsRegex = /^(ws(s)?):\/\//i;
```

Comment:
Minor, but perhaps we can centralize this regex in a constant variable near the top of the module to keep things a tad DRYer?

Reply:
Agreed and moved constant to top of the file